src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchForm = ({ api, method, jsonBody, onChange, onSubmit }) => (
+  <form onSubmit={onSubmit}>
+    <input type="text" name="api" placeholder="API URL" value={api} onChange={onChange} />
+
+    <label>
+      <input type="radio" name="method" value="get" checked={method === 'get'} onChange={onChange}/>
```

Comment:
Your app wasn't working because all of your methods need to be capitalized in both the value and the checked.
So value="GET" checked={method==="GET}